### PR TITLE
Update display /order in the doc

### PR DESCRIPTION
**Pull request by pthegner**
# Description

This PR update some display in the doc

## Issue

Issue link: BOT-37, BOT-24

### CODE REVIEW

**Comment by vincentlepot** [comment]
For the record, it would be great to have an API documentation in Swagger / OpenAPI format. It would be more usable

File: documentation/disruptions.md [hunk 1]
```diff
@@ -70,2882 +98,3052 @@ Chaos uses some concepts, here are the essentials.
 | X-Contributors       | contributor code. A contributor is owner of a disruption                       | true     |                         |
 | X-Coverage           | coverage of navitia services                                                   | true     |                         |
 
-# List of disruptions [/disruptions]
```
Comment (vincentlepot) [inline documentation/disruptions.md]: Why move disruptions at the bottom? It's the more important API in Chaos.

Comment (pthegner) [inline documentation/disruptions.md]: It's in the order of required (on POST) you should have already a cause set, and tags and properties (optional). If you want, i could set some link for each part (like a summary) .

Comment (vincentlepot) [inline documentation/disruptions.md]: It would be nice. In fact, I believe causes, severities, tags,... are usually configured by us. But I may be wrong.

Comment (pthegner) [inline documentation/disruptions.md]: I add it, but i don't know if i keep the order of usage for the summary, or if i changed it to be like the order of the document.

**Pull request merged**